src/taipy/core/_manager/_manager.py

```diff
@@ -122,3 +122,10 @@ def _export(cls, id: str, folder_path: Union[str, pathlib.Path]):
         Export an entity.
         """
         return cls._repository._export(id, folder_path)
+
+    @classmethod
+    def _get_all_entities_by_config_id(cls, config_id: str) -> List[EntityType]:
```

Comment:
Maybe we can simplify the name. Here are a few proposal.
`_get_by_config_id` (my preference)
`_get_all_by_config_id`
`_get_entities_by_config_id`

Reply:
I'm going with `_get_by_config_id` :D